### PR TITLE
Adds UI Icon for lay/get up (crawling)

### PR DESCRIPTION
**Pull request by Birdtalon**
Relevant code adding UI icon for laying down / getting up.

Sprites needed for all the UI styles. Using my codersprite for midnight UI at the moment.

Location:

dreamseeker_2017-09-13_18-48-10

### CODE REVIEW

File: code/_onclick/hud/screen_objects.dm [hunk 1]
```diff
@@ -57,11 +57,23 @@
...
+	if(isobserver(usr))
+		return
+	M.lay_down()
+
 /obj/screen/craft
 	name = "crafting menu"
 	icon = 'icons/mob/screen_midnight.dmi'
 	icon_state = "craft"
-	screen_loc = ui_crafting
+	screen_loc = ui_lay
```
Comment (FlattestGuitar) [inline code/_onclick/hud/screen_objects.dm]: :eyes:

Comment (Birdtalon) [reply code/_onclick/hud/screen_objects.dm]: Oh that explains why that part wasn't working right.

File: code/_onclick/hud/screen_objects.dm [hunk 2]
```diff
@@ -57,11 +57,23 @@
...
 
+/obj/screen/lay
+	name = "lay down"
+	icon = 'icons/mob/screen_midnight.dmi'
+	icon_state = "lay"
+	screen_loc = ui_crafting
+
+/obj/screen/lay/Click()
+	var/mob/living/M = usr
+	if(isobserver(usr))
```
Comment (FlattestGuitar) [inline code/_onclick/hud/screen_objects.dm]: Do an `istype(M)` check here instead.

**Comment by FlattestGuitar** [comment]
Not a fan of the icon's location and size. In my opinion it should be next to the drop/throw stuff and the size of *at least* the vertical buttons, if not just how big an inventory slot is.

Also, toggling the sprite depending on whether you're trying to lie or stand would be great, so that you're able to tell whether you're resting or on the ground because of a stun.

**Comment by SamHPurp** [comment]
Conflicts - also, hows the sprites coming on? Someone committed to taking this on?

**Comment by Birdtalon** [comment]
I'll fix the conflicts once we actually get a sprite to add. Also I'll have to review this PR anyway as the PDA slot changes the location of the icons.

**Comment by Birdtalon** [comment]
Closing this as my repo broke and I can't seem to push to this any more. New one will follow.

**Pull request closed without merging**